### DISCUSSION TITLE
Messages status changed from 'ready' to 'unacked'

### DISCUSSION THREAD

**Discussion by vipin6987** [rabbitmq/rabbitmq-server#9864] [Other] [answered]
My concern is that when I am inserting N numbers of messages in the queue, and then consume the messages from queue the status of all the messages changed from 'ready' to 'unacked'. Could you kindly provide me with a solution to resolve this issue?

**Comment by mkuratczyk** [comment]
That's the basics of how RabbitMQ works, covered in the second tutorial:
https://www.rabbitmq.com/tutorials/tutorial-two-go.html

Unacked message is a message that was delivered to a consumer but wasn't acknowledged by that consumer yet.

**Reply by vipin6987** [reply, marked as answer]
I know the basics, actually my binary isn't working properly because my developer wrote the wrong code. Thanks for your support Jai Shree Ram .
